cmd/thanos/query.go

```diff
@@ -70,6 +70,8 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application) {
 	maxConcurrentQueries := cmd.Flag("query.max-concurrent", "Maximum number of queries processed concurrently by query node.").
 		Default("20").Int()
 
+	lookbackDelta := cmd.Flag("query.lookback-delta", "The maximum lookback duration for retrieving metrics during expression evaluations. For example with a look back of 7min and a scrape interval of 10min, promql will fill in the gap only for 7min and will show the other 3min as a gap. Should be set to at least 2 times the slowest scrape interval.").Duration()
```

Comment:
Awesome, this is much better! 

We can be clear what it means in details and also add default here! (: 

```suggestion
	lookbackDelta := cmd.Flag("query.lookback-delta", "The maximum lookback duration for retrieving metrics during expression evaluations. PromQL always evaluates the query for the certain timestamp (query range timestamps are deduced by step). Since scrape intervals might be different, PromQL looks back for given amount of time to get latest sample. If it exceeds the maximum lookback delta it assumes series is stale and returns none (a gap). This is why lookback delta should be set to at least 2 times of the slowest scrape interval.").Default("5m").Duration()
```

WDYT?